### ISSUE TITLE
Allow customization of lane-changing urgency (trac #1145)

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#1145]
Right now vehicles do not perform cooperative changes for vehicles which try to change for speedGain (only strategic and TraCi changes are sufficiently urgen). This is contrary to behavior observable in reality and should be made possible.

related to #1136

Migrated from http://sumo.dlr.de/ticket/1145
```json
{
    "status": "new", 
    "changetime": "2016-05-23T10:32:57Z", 
    "description": "Right now vehicles do not perform cooperative changes for vehicles which try to change for speedGain (only strategic and TraCi changes are sufficiently urgen). This is contrary to behavior observable in reality and should be made possible.\n\nrelated to #1136", 
    "reporter": "namdre", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1463999577740321", 
    "component": "microsim - lane change", 
    "summary": "Allow customization of lane-changing urgency", 
    "priority": "major", 
    "keywords": "", 
    "time": "2014-02-12T07:21:23Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by behrisch** [comment]
@namdre This should be possible with the lc parameters?

**Comment by namdre** [comment]
@behrisch Not quite. While all motivations can be configured in regard to their strength / reaction speed, there is still a distinction between urgent and non-urgent changing:
* For urgent changing (strategic, TraCI), the ego vehicles and its neighbors perform precise cooperative speed adaptions
* For non-urgent changing (cooperative, speedGain, keepRight, sublane), this is not done (except for some very coarse stuff that still dates from the DK2008 era).

This ticket is about allowing configurable urgent changes for all motivations

**Comment by namdre** [comment]
Idea: Since the internal "motivation" values `mySpeedGainProbability` and `myKeepRightProbability` can grow arbitrarily strong, provide an new parameter that sets a threshold value for enabling urgency: lcSpeedGainUrgency, lcKeepRightUrgency

**Comment by namdre** [comment]
just came up again with @robertalms  w.r.t motorway calibration
